backend/src/database/seeds/01_capabilities.js

```diff
@@ -127,6 +127,18 @@ const capabilities = [
     id: 32,
     name: "project_status_read_mine",
   },
+  {
+    id: 33,
+    name: "subcontractors_read_mine",
+  },
+  {
+    id: 34,
+    name: "subcontractors_update_all",
+  },
+  {
+    id: 35,
+    name: "subcontractors_update_mine",
+  },
```

Comment:
Probably outside the scope of this task but I wonder if we should even be specifying ids here or whether we can just set it up to use the array index as the id since we only actually care about the name. Just thinking because this is going to cause a bunch of merge conflicts since we'll all be adding our own capabilities with the same ids.

Reply:
I had pause over this too, actually - because it's also a pain if you have an urge to re-order the list to make it more organized. :)
To be fair, it wouldn't normally be a problem though, and shouldn't be _that_ big a deal to rebase...(he says, having already merged his changes).

Thanks for bringing this up. I think what you say has merit and could warrant a discussion - you could always create a ticket for it and then argue for it. I don't think it fits under the definition of MVP for the existing ticket.